x-pack/test_serverless/functional/test_suites/observability/discover/context_awareness/_get_app_menu.ts

```diff
@@ -25,7 +25,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
   describe('extension getAppMenu', () => {
     before(async () => {
-      await svlCommonPage.loginAsViewer();
```

Comment:
Does it hide the menu item for the viewer now? Might be good to have a separate test for it.